lib/display.rb

```diff
@@ -0,0 +1,42 @@
+class Display
+  CONFIRMATION_PROMPT   = 'Confirm order? Hit ⏎ to confirm, c to cancel'
+  INVALID_MEAL_MESSAGE  = ' is not a meal. Please make a valid choice'
+  MEAL_ADD_CONFIRMATION = ' has been added to your order 🍕'
```

Comment:
Really good to have an extra step to confirm the order, and a confirmation message that an item had been added.